### PR TITLE
add section on markdown description properties

### LINKED ISSUE #4305: Add documentation on markdown descriptions for properties
## What type of issue is it? (Choose one - delete the others)
Missing documentation

## Describe the issue

With Umbraco 9.2+ it is now possible to write markdown + a few custom things (collapse) in the property descriptions. I was looking for documentation on that, but couldn't find any.

It is somewhat documented on the [PR](https://github.com/umbraco/Umbraco-CMS/pull/11628).

I'd be up for submitting a PR, but honestly not sure where to put it - maybe somewhere on [this page](https://our.umbraco.com/documentation/Fundamentals/Data/Defining-content/)?

### PR DESCRIPTION
**Pull request by jemayn**
This pr closes #4305

### CODE REVIEW

**Comment by jemayn** [comment]
I can't really tell why the Vale linter is failing, seems it's only showing a few warnings before it quits 🤷‍♂️

File: Fundamentals/Data/Defining-content/index-v8.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+Go back to the __Settings section__ and create a new Document Type and name it "Text Page". Add a group called "Content"
+and this time we'll add two properties. First, make a property called Summary using the __Textarea__ editor and secondly create a property called "Body Text" and reuse the __Rich Text Editor__ Data Type.
+
+### Creating child nodes
+
+Before we can create a Text Page in the __Content__ section, we need to allow the Text Page Document Type to be created as a child node to the Home node. Select the Home Document Type and go to the __Permissions__ group. Click __Add child__ and select Text Page.
+
+![Allowing child nodes](images/v8Screenshots/setPagePermissions.png)
+
+Go to the __Content__ section and click the menu icon (•••) next to the *Home* node and select the Text page Document Type. We'll name the page "About us". We now have a very basic content structure.
```
Comment (sofietoft) [inline Fundamentals/Data/Defining-content/index-v8.md:152]:
```suggestion
Go to the __Content__ section and click the menu icon (•••) next to the *Home* node and select the Text page Document Type. We'll name the page "About us". We now have a basic content structure.
```

File: Fundamentals/Data/Defining-content/index-v8.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+![Basic content structure](images/v8Screenshots/createAboutUs.png)
+
+Document Types are very flexible and can be used in a myriad of ways from defining a piece of reusable content or an entire page, to acting as a container or repository.
```
Comment (sofietoft) [inline Fundamentals/Data/Defining-content/index-v8.md:156]:
```suggestion
Document Types are flexible and can be used in a myriad of ways from defining a piece of reusable content or an entire page, to acting as a container or repository.
```

**Review by sofietoft — COMMENTED** [review]
Think I found the things Vale complained about.
Looks like a couple of "very" had found their way into the article 😄

**Comment by jemayn** [comment]
@sofietoft 

As this new markdown syntax was added from 9.2+ I had to create a new version - weird thing is the issues were on the original article (now called index-v8.md) 🤷‍♂️

Also seems strange it fails entirely if they are only at the warning level and not error - but it looks like it works now - thanks for the help! 😊

**Comment by sofietoft** [comment]
Yeah, we had to revamp the role vale thing some time back, because the old system had stopped working for some strange reason 🤷 
So now it will say it failed even if it's just warnings 😅 

Looks like it's happy now, though! Will merge it in 💪 Thanks!

**Pull request merged**